build/configs/rtl8720e/hello/defconfig

```diff
@@ -141,12 +141,12 @@ CONFIG_INIC_IPC_HIGH_TP=1
 #
 # Realtek RTL8720E BLE Support
 #
-#CONFIG_AMEBALITE_BLE=y
-#CONFIG_FTL_ENABLED=y
+CONFIG_AMEBALITE_BLE=y
```

Comment:
hello is minimum set of TizenRT, so I think you should not enable BLE here.

Reply:
Hi Mr Kwon, I enable BLE here because I disabled it in my previous PR.
Actually for TP1x, I just checked, we also enabled BLE in hello config.
So do you want me to keep this PR's defconfig change, or disable BLE?
Thank you!